test/IconFontTester/IconFontTester/MainPage.xaml.cs

```diff
@@ -0,0 +1,104 @@
+using System.Collections.ObjectModel;
+using System.Text;
+using Newtonsoft.Json;
+using Uno.IconFontTester;
+using Uno.IconFontTester.Models;
+using Windows.System;
+
+namespace IconFontTester;
+
+public sealed partial class MainPage : Page
+{
+    private string _searchTerm;
+
+    public MainPage()
+    {
+        this.InitializeComponent();
+    }
+
+    protected override async void OnNavigatedTo(NavigationEventArgs e)
+    {
+        base.OnNavigatedTo(e);
+        Refresh();
+    }
+
+    public string SearchTerm
+    {
+        get => _searchTerm;
+        set
+        {
+            _searchTerm = value;
+            UpdateSearch();
+        }
+    }
+
+    private List<IconItem> _allGlyphs = new List<IconItem>();
+
+    public ObservableCollection<IconItem> FilteredGlyphs { get; } = new ObservableCollection<IconItem>();
+
+    public async void OpenSelectionFolder()
+    {
+        var folder = await ApplicationData.Current.LocalFolder.CreateFolderAsync("UnoIconTest", CreationCollisionOption.OpenIfExists);
+        await Launcher.LaunchFolderAsync(folder);
```

Comment:
@MartinZikmund I tested the application locally and I am getting some exceptions when trying to use the "Open selection folder" button.

For Desktop for example:
```
Exception thrown: 'System.NotImplementedException' in Uno.dll
An exception of type 'System.NotImplementedException' occurred in Uno.dll but was not handled in user code
The member IAsyncOperation<bool> Launcher.LaunchFolderAsync(IStorageFolder folder) is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=IAsyncOperation%3Cbool%3E%20Launcher.LaunchFolderAsync%28IStorageFolder%20folder%29
```

Reply:
Also "Open selection folder" button does not do anything for WASM. For more info, I just checked out your branch, restored packages, and launched the app without changing anything else. 
If you can double-check all platforms on your side just in case, please.